blocks/bricks/recurrent.py

```diff
@@ -439,9 +439,10 @@ class GatedRecurrent(BaseRecurrent, Initializable):
 
     Parameters
     ----------
-    activation : :class:`.Brick`
-        The brick to apply as activation.
-    gated_activation : :class:`.Brick` or None
+    activation : :class:`.Brick` or None
```

Comment:
Can you please permute the entries of `Parameters` section in the same order as argument of the `__init__` method?